crates/subspace-farmer/src/bin/subspace-farmer/main.rs

```diff
@@ -125,7 +125,7 @@ struct DsnArgs {
     bootstrap_nodes: Vec<Multiaddr>,
     /// Multiaddr to listen on for subspace networking, for instance `/ip4/0.0.0.0/tcp/0`,
     /// multiple are supported.
-    #[arg(long, default_value = "/ip4/0.0.0.0/tcp/30533")]
+    #[arg(long, default_value = "/ip4/0.0.0.0/udp/30533/quic-v1")]
```

Comment:
I do not think this is correct either, it should listen on both TCP and UDP, shouldn't it? How would you connect to anyone with UDP otherwise? Also you probably want to update bootstrap node addresses in chain spec for devnet in this case.